README.md

```diff
@@ -1,19 +1,20 @@
 [![AsyncAPI Design System](/assets/github-repobanner-designsystem.png)](https://www.asyncapi.com)
 
-## Environment Setup
+  # :deciduous_tree:Environment Setup
 
-### How to setup storybook locally
 
-- Before starting the Storybook environment locally, install all the required dependencies. Run the command below to install the dependencies: 
+### :joystick:How to setup storybook locally
+
+- Before starting the Storybook environment locally, install all the required dependencies. Run the command below to install the dependencies:
 
 ```cmd
 npm install --save-dev
```

Comment:
Hmmm 🤔  It should be normal `npm install`.

Reply:
Ok.